app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -28,6 +28,7 @@ class SearchFragment : Fragment() {
     private lateinit var rootView: View
     private var loadEventsAgain = false
     private lateinit var searchView: SearchView
+    private var isSearched = false
```

Comment:
Remove this variable from here, react to changes from viewmodel only

Reply:
Updated